test/blackbox-tests/test-cases/pkg/env-update-prepend-sep.t

```diff
@@ -0,0 +1,34 @@
+We have 3 opam packages depending on each other in order. We test how setenv interacts
+with the := operator.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+  $ mkenvtestpkg package1 "" :=
+  $ mkenvtestpkg package2 package1 :=
+  $ mkenvtestpkg package3 package2 :=
```

Comment:
how about printing the relevant command after making the package? Otherwise, it's actually hard to understand what the test is doing.

Reply:
Printing the command trace is quite verbose, therefore I will just echo what it is doing in a readable form.

Reply:
Yeah, you don't need the entire command. Just the subset relevant for the test. So the line that updates the environment basically.